src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3641,6 +3647,19 @@ float ViewProviderSketch::getScaleFactor()
     }
 }
 
+void ViewProviderSketch::InitItemsSizes()

```

Comment:
Basically, this function returns `coinFontSize` and `constraintIconSize`.

Maybe split it into two functions:

```
int ViewProviderSketch::getCoinFontSizePx();
int ViewProviderSketch::getConstraintIconSizePx();
```
Are there any performance issue that justify caching? Consider the scenario where device pixel ratio changes on the fly (i.e. when application window is moved to another display).